api/src/functions/express/app.ts

```diff
@@ -282,7 +282,7 @@ const apiFormationClient = ApiFormationClient(
   logger
 );
 
-const apiFormationHealthCheckClient = apiFormationClient.health;
+const webServiceFormationHealthCheckClient = apiFormationClient.health;
```

Comment:
[sand] In our codebase `webService` == `Boomi` but we don't officially endorse Boomi as a technology so our code should not reflect that. I'd prefer either go back to `apiFormation...` or `formationAPI...` or something else.

Reply:
done